### PR TITLE
Filtering

### PR DESCRIPTION
**Pull request by TGlide**

### CODE REVIEW

File: src/routes/console/project-[project]/settings/migrations/(import)/step1.svelte [hunk 1]
```diff
@@ -57,13 +93,30 @@
...
+                    class="card is-allow-focus u-height-100-percent u-flex u-flex-vertical u-cursor-pointer u-margin-block-start-24">
+                    <div class="content">
+                        <p>{project.displayName}</p>
+                        <span>{project.projectId}</span>
+                    </div>
+                </div>
+            {/each}
 
+            <p class="u-text-center u-margin-block-start-24">
+                Signed in as test@test.com <button
```
Comment (ArmanNik) [inline src/routes/console/project-[project]/settings/migrations/(import)/step1.svelte:115]: Does `test@test.com` need to stay here?

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/filters.svelte [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    }
+
+    hr {
+        height: 1px;
+        width: calc(100% + 2rem);
+        background-color: hsl(var(--color-border));
+
+        margin-block-start: 1rem;
+        margin-inline: -1rem;
+    }
```
[the remark this answers is not in the document]
Comment (TGlide) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/filters.svelte:307]: Didn't find them on pink.appwrite.io 🤔

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.svelte [hunk 3]
```diff
@@ -105,3 +108,15 @@
...
+    .cool-grid {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        gap: 2rem;
+
+        > :global(h2) {
+            grid-column: span 2;
+        }
+    }
+</style>
```
[the remark this answers is not in the document]
Comment (ArmanNik) [reply src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.svelte]: Yes :)

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.ts [hunk 4]
```diff
@@ -4,21 +4,36 @@ import { sdk } from '$lib/stores/sdk';
...
         limit,
         view,
         documents: await sdk.forProject.databases.listDocuments(
             params.database,
             params.collection,
-            [Query.limit(limit), Query.offset(offset), Query.orderDesc('$createdAt')]
+            [
+                Query.limit(limit),
+                Query.offset(offset),
+                Query.orderDesc('$createdAt'),
```
Comment (christyjacob4) [inline src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.ts:34]:
```suggestion
                Query.orderDesc(''),
```
we have some ongoing index issues with using the $createdAt attribute in ordering . So we use the default ordering based on the internal IDs

File: src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.ts [hunk 5]
```diff
@@ -4,21 +4,36 @@ import { sdk } from '$lib/stores/sdk';
...
 export const load: PageLoad = async ({ params, depends, url, route }) => {
     depends(Dependencies.DOCUMENTS);
     const page = getPage(url);
     const limit = getLimit(url, route, PAGE_LIMIT);
     const view = getView(url, route, View.Grid);
     const offset = pageToOffset(page, limit);
 
+    const queries = url.searchParams.get('query');
+    const parsedQueries = queryParamToMap(queries || '[]');
+    console.log(parsedQueries);
```
Comment (christyjacob4) [inline src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/+page.ts:22]: leftover ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**